### DISCUSSION TITLE
I found that I was fined. How do I know the reason

### DISCUSSION THREAD

**Discussion by hsn999** [filecoin-project/lotus#5485] [Storage Provider] [answered]
I found that I was fined. How do I know the reason。I can check penalty on filfox,but there is no reason in it.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
run

`lotus-miner proving deadlines` and see if there are faulty sectors - if yes you missed 2 windowedPosts in a row
